### PR TITLE
Adding all the new predictions

### LINKED ISSUE #16: Create a dashboard for the new PLparameters.pdf
PLparameters.pdf has new predictions, we should show them in another dashboard. And perhaps show them also for the countries with old predictions as well, for comparison.

### PR DESCRIPTION
**Pull request by lukipuki**

### CODE REVIEW

File: covid_graphs/covid_graphs/formula.py
Comment (miskosz) [inline covid_graphs/covid_graphs/formula.py]: Nit: a curve

File: covid_graphs/covid_graphs/formula.py
Comment (miskosz) [inline covid_graphs/covid_graphs/formula.py:41]: Did you intentionally change `>` to `>=`?

File: data/download_all.zsh
Comment (miskosz) [inline data/download_all.zsh]: You are using `US` here and not `United States`, not sure if intentional.

**Comment by lukipuki** [comment]
FYI, there's something incorrect about the predictions - I found inconsistencies between us and Katka/Rišo for Croatia and Czechia. They are investigating. Until then, we shouldn't merge this.

**Review by miskosz — APPROVED** [review]
LGTM

**Comment by miskosz** [comment]
@lukipuki I'd suggest commenting Croatia and Czechia out and merging like that.

We are not displaying the graphs anyway, are we?

File: covid_graphs/covid_graphs/formula.py
Comment (lukipuki) [reply covid_graphs/covid_graphs/formula.py:41]: Yes, that's how Katka does the fitting. I spotted in on Lithuania, where the difference is 12 days :smile:

**Comment by miskosz** [comment]
Oh, those two were just examples, weren't they :D

File: data/download_all.zsh
Comment (lukipuki) [reply data/download_all.zsh]: Thanks for noticing, that's a bug.

**Comment by lukipuki** [comment]
Yes, those are just examples, I suspect a bigger issue somewhere (could be on either side).

But you are right, let's merge and we'll wait for their comments.

File: covid_graphs/covid_graphs/formula.py
Comment (lukipuki) [reply covid_graphs/covid_graphs/formula.py]: Done

**Pull request merged**
